pkg/operator/apis/aro.openshift.io/v1alpha1/cluster_types.go

```diff
@@ -96,6 +99,7 @@ type ClusterStatus struct {
 	OperatorVersion   string                         `json:"operatorVersion,omitempty"`
 	Conditions        []operatorv1.OperatorCondition `json:"conditions,omitempty"`
 	RedHatKeysPresent []string                       `json:"redHatKeysPresent,omitempty"`
+	AdvisorConditions []operatorv1.OperatorCondition `json:"advisorConditions,omitempty"`
```

Comment:
I haven't reviewed the whole PR yet, but so far I'm not convinced that we need to introduce `advisorConditions`:

1. I've never seen `advisorConditions` in other operators. I would rather be consistent with rest of them. It would never occur to me to look into something else if there is `conditions` in the operator. 
2. Other operators seem to report similar kind of issues to `conditions`. We don't have to alert on all of the conditions, if some of them are just informative.